reporterdashboard/dashboard.py

```diff
@@ -0,0 +1,112 @@
+import json
+import argparse
+
+
+def get_matching_test_cases(test_cases_1, test_cases_2):
+    """Returns test cases that are both in test_cases_1 and test_cases_2.
+
+    Test cases are ordered by their name.
+    """
+    # create sorted copies
+    sorted_1 = sorted(test_cases_1, key=lambda d: d["name"])
+    sorted_2 = sorted(test_cases_2, key=lambda d: d["name"])
+    overlapping_test_cases = []
+
+    while True:
```

Comment:
Is there not some python utility for this? Otherwise, this could use a comment what the algorithm is, because a `while True`: feels a bit dangerous